frontend/src/main.js

```diff
@@ -14,16 +14,14 @@ import VJsoneditor from 'v-jsoneditor'
 import MarkdownItVue from 'markdown-it-vue'
 
 
-
-// Import Bootstrap an BootstrapVue CSS files (order is important)
-import 'bootstrap/dist/css/bootstrap.css'
-import 'bootstrap-vue/dist/bootstrap-vue.css'
-
-
 //Importing scss assets needs an @/ in front of it
 
 import "@/assets/sass/app.scss"
 
+// Import Bootstrap an BootstrapVue CSS files (after app.scss, which includes bootstrap)
+import 'bootstrap-vue/dist/bootstrap-vue.css'
```

Comment:
I've changed `app.scss` to import the bootstrap sass directly, so we don't need to import the pre-compiled CSS as well.